app/boards/shields/pteron36/pteron36.dtsi

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020 Pete Johanson
```

Comment:
Might want to fix the copyright headers in this file and others to `Copyright (c) 2022 The ZMK Contributors` as well.